### PR TITLE
Add Arturo's suggestions (WIP)

### PR DESCRIPTION
**Pull request by ismayc**
Only for reviewing suggested Chapter 7 edits at this point. No review necessary just yet.

### CODE REVIEW

File: 07-sampling.Rmd [hunk 1]
```diff
@@ -73,128 +70,111 @@ percent_red_chr <- prop_red %>% percent(accuracy = 0.1)
...
-
-Let's now ask ourselves, what proportion of this bowl's balls are red?
+Take a look at the bowl in Figure \@ref(fig:sampling-exercise-1). It has a certain number of red balls and a certain number of white balls all of equal size. `r if_else(is_latex_output(), '(Note that in this printed version of the book "red" corresponds to the darker-colored balls, and "white" corresponds to the lighter-colored balls. We kept the reference to "red" and "white" throughout this book since those are the actual colors of the balls as seen in the background of the image on our book\'s [cover](https://moderndive.com/images/logos/book_cover.png).)', '')` The balls in the bowl have been mixed beforehand, as there does not seem to be any coherent pattern to the spatial distribution of the red and white balls. Our task is to determine, at least with some level of precision, the proportion of red balls in the bowl.
```
Comment (ismayc) [inline 07-sampling.Rmd:73]: "spatial distribution" seems a bit too complex for the target audience here. Maybe something like "pattern to how the red and white balls fall in the hopper"?

Comment (ismayc) [inline 07-sampling.Rmd:73]: I'm a little worried with the use of the word "precision" here as well.

File: 07-sampling.Rmd [hunk 2]
```diff
@@ -73,128 +70,111 @@ percent_red_chr <- prop_red %>% percent(accuracy = 0.1)
...
 
 ```{r sampling-exercise-1, echo=FALSE, fig.cap="A bowl with red and white balls.", purl=FALSE, out.width = "95%", purl=FALSE}
 include_graphics("images/sampling/balls/sampling_bowl_1.jpg")
 ```
 
-One way to answer this question would be to perform an exhaustive count: remove each ball individually, count the number of red balls and the number of white balls, and divide the number of red balls by the total number of balls. However, this would be a long and tedious process. 
+One way to do this is to perform an exhaustive count: remove each ball individually, count the number of red balls and divide the number of red balls by the total number of balls. Doing this carefully can determine the exact proportion of red balls in the bowl but it requires a long and tedious process.
```
Comment (ismayc) [inline 07-sampling.Rmd:79]:
```suggestion
One way to do this is to perform an exhaustive count: remove each ball individually, count the number of red balls and divide the number of red balls by the total number of balls. Doing this carefully can determine the exact proportion of red balls in the bowl, but it requires a long and tedious process.
```

File: 07-sampling.Rmd [hunk 3]
```diff
@@ -73,128 +70,111 @@ percent_red_chr <- prop_red %>% percent(accuracy = 0.1)
...
 
 
-### Using the shovel once 
 
-Instead of performing an exhaustive count, let's insert a shovel into the bowl as seen in Figure \@ref(fig:sampling-exercise-2). Using the shovel, let's remove $5 \cdot 10 = 50$ balls, as seen in Figure \@ref(fig:sampling-exercise-3).
+### One sample
+
+Instead of performing an exhaustive count, we take a sample of balls. We do this by inserting a shovel into the bowl as shown in Figure \@ref(fig:sampling-exercise-2) and taking a sample of $5 \cdot 10 = 50$ balls as shown in Figure \@ref(fig:sampling-exercise-3).
```
Comment (ismayc) [inline 07-sampling.Rmd:85]: We haven't used the word "sample" here yet intentionally. Since that's an important concept, we'd rather build on the ideas and then introduce terminology later.

File: 07-sampling.Rmd [hunk 4]
```diff
@@ -73,128 +70,111 @@ percent_red_chr <- prop_red %>% percent(accuracy = 0.1)
...
-However, say, we started this activity over from the beginning. In other words, we replace the 50 balls back into the bowl and start over. Would we remove exactly 17 red balls again? In other words, would our guess at the proportion of the bowl's balls that are red be exactly 34% again? Maybe? 
 
-What if we repeated this activity several times following the process shown in Figure \@ref(fig:sampling-exercise-3b)? Would we obtain exactly 17 red balls each time? In other words, would our guess at the proportion of the bowl's balls that are red be exactly 34% every time? Surely not. Let's repeat this exercise several times with the help of 33 groups of friends to understand how the value differs with repetition.
 
 
-### Using the shovel 33 times {#student-shovels}
+### Thirty-three samples {#student-shovels}
 
-Each of our 33 groups of friends will do the following: 
+To understand why this work well, we repeat this activity many times as shown in Figure \@ref(fig:sampling-exercise-3b). Each time we do the following:
```
Comment (ismayc) [inline 07-sampling.Rmd:103]:
```suggestion
To understand why this works well, we repeat this activity many times as shown in Figure \@ref(fig:sampling-exercise-3b). Each time we do the following: 
```

File: 07-sampling.Rmd [hunk 5]
```diff
@@ -73,128 +70,111 @@ percent_red_chr <- prop_red %>% percent(accuracy = 0.1)
...
 include_graphics("images/sampling/balls/sampling_bowl_3_cropped.jpg")
 ```
 
-Observe that 17 of the balls are red and thus 0.34 = 34% of the shovel's balls are red. We can view the proportion of balls that are red in this shovel as a guess of the proportion of balls that are red in the entire bowl. While not as exact as doing an exhaustive count of all the balls in the bowl, our guess of 34% took much less time and energy to make. 
+Observe that 17 of the balls are red and thus the proportion of red balls in the sample is 17/50 = 0.34 or 34%. We call this the _sample proportion_ of red balls. The sample proportion is a guess of the proportion of red balls in the entire bowl. It is likely that the proportion of red balls in the entire bowl is not exactly 34% but, as we show later, this is probably a good guess and it took much less time and energy to obtain.
 
-However, say, we started this activity over from the beginning. In other words, we replace the 50 balls back into the bowl and start over. Would we remove exactly 17 red balls again? In other words, would our guess at the proportion of the bowl's balls that are red be exactly 34% again? Maybe?
```
Comment (ismayc) [inline 07-sampling.Rmd:107]: Not sure how I feel about removing these questions here and replacing it with "not surprisingly" later. Remember the target audience here is people that have little to no mathematical background.

File: 07-sampling.Rmd [hunk 6]
```diff
@@ -73,128 +70,111 @@ percent_red_chr <- prop_red %>% percent(accuracy = 0.1)
...
+
+```{r sampling-exercise-4, echo=FALSE, fig.cap="Students drawing a histogram of proportions.", purl=FALSE, out.width = "80%"}
 include_graphics("images/sampling/balls/tactile_3_a.jpg")
 ```
 
-Recall from Section \@ref(histograms) that histograms allow us to visualize the *distribution* \index{distribution} of a numerical variable. In particular, where the center of the values falls and how the values vary. A partially completed histogram of the first 10 out of 33 groups of friends' results can be seen in Figure \@ref(fig:sampling-exercise-5).
+Recall from Section \@ref(histograms) that histograms help us visualize the *distribution* \index{distribution} of a numerical variable. In particular, where the center of the values falls and how the values vary. A histogram of the first 10 sample proportions can be seen in Figure \@ref(fig:sampling-exercise-5).
```
Comment (ismayc) [inline 07-sampling.Rmd:120]: This is another spot where I hesitate to just use the phrase "sample proportions" here while we are building up the concepts. I want to make sure the cognitive load is not too much for students. If you have a good reason to switch it up beyond just brevity, I'm all ears.

File: 07-sampling.Rmd [hunk 7]
```diff
@@ -73,128 +70,111 @@ percent_red_chr <- prop_red %>% percent(accuracy = 0.1)
...
+The activity introduced the concept of \index{sampling} *sampling*. We wanted to know the proportion of red balls in a bowl. Since the bowl had many balls, performing an exhaustive count of red and white balls was impractical. Instead, we first took a *sample* of 50 balls and determine that the *sample proportion* was 34%. This is an *estimate* of the proportion of red balls in the entire bowl. We then took more samples and obtain more sample proportions, 33 in total. This activity is what in statistics we call *sampling*. 
 
-The purpose of this sampling activity was to develop an understanding of two key concepts relating to sampling: 
+Since we returned the observed balls every time before taking a new sample we were *sampling with replacement*, and since we mixed the balls before taking each sample and then used a shovel, the samples were *randomly drawn*. When this happens we call them *random samples*. As shown in Figure \@ref(fig:samplingdistribution-tactile), different random samples produce different sample proportions. This phenomenon is called *sampling variation* and it is a central concept in this chapter. \index{sampling!variation}
```
Comment (ismayc) [inline 07-sampling.Rmd:164]:
```suggestion
Since we returned the observed balls every time before taking a new sample we were *sampling with replacement*. Since we mixed the balls before taking each sample and then used a shovel, the samples were *randomly drawn*. When this happens we call them *random samples*. As shown in Figure \@ref(fig:samplingdistribution-tactile), different random samples produce different sample proportions. This phenomenon is called *sampling variation* and it is a central concept in this chapter. \index{sampling!variation}
```

File: 07-sampling.Rmd [hunk 8]
```diff
@@ -73,128 +70,111 @@ percent_red_chr <- prop_red %>% percent(accuracy = 0.1)
...
 
-1. Understanding the effect of sampling variation.
-1. Understanding the effect of sample size on sampling variation. 
+In Section \@ref(sampling-simulation), we extend the sampling activity presented here to sampling done via simulations on a computer. The use of simulations permits to study *sampling variation* not only for 33 random samples but many more, often thousands, tens of thousands, or even more. In addition, we do not need to restrict the samples to 50 balls, we can change the size of the virtual shovel to any size. As we show in the next section, the sample size of the samples taken has a direct effect on the magnitude of the *sampling variation*.
```
Comment (ismayc) [inline 07-sampling.Rmd:166]:
```suggestion
In Section \@ref(sampling-simulation), we extend the sampling activity presented here to sampling done via simulations on a computer. The use of simulations permits us to study *sampling variation* not only for 33 random samples but many more, often thousands, tens of thousands, or even more. In addition, we do not need to restrict the samples to 50 balls, we can change the size of the virtual shovel to any size. As we show in the next section, the sample size of the samples taken has a direct effect on the magnitude of the *sampling variation*.
```

File: 07-sampling.Rmd [hunk 9]
```diff
@@ -323,29 +300,27 @@ virtual_prop_red <- virtual_samples %>%
 virtual_prop_red
 ```
 
-As with our 33 groups of friends' tactile samples, there is variation in the resulting 33 virtual proportions red. Let's visualize this variation in a histogram in Figure \@ref(fig:samplingdistribution-virtual). Note that we add `binwidth = 0.05` and `boundary = 0.4` arguments as well. Recall that setting `boundary = 0.4` ensures a binning scheme with one of the bins' boundaries at 0.4. Since the `binwidth = 0.05` is also set, this will create bins with boundaries at 0.30, 0.35, 0.45, 0.5, etc. as well.
+As it was the case in the tactile activity, there is variation in the resulting 33 proportions from virtual samples. As shown in Figure \@ref(fig:samplingdistribution-virtual) a histogram helps us visualize this sampling variation. As we did in Section \@ref{#sampling-activity}, we construct the histogram using `ggplot()`, `geom_histogram()`, and including the arguments `binwidth = 0.05` and `boundary = 0.4`. This creates the appropriate bins with boundaries at 0.3, 0.35, 0.4, 0.45, etc., as shown below:
```
Comment (ismayc) [inline 07-sampling.Rmd:303]:
```suggestion
As was the case in the tactile activity, there is variation in the resulting 33 proportions from virtual samples. As shown in Figure \@ref(fig:samplingdistribution-virtual) a histogram helps us visualize this sampling variation. As we did in Section \@ref{#sampling-activity}, we construct the histogram using `ggplot()`, `geom_histogram()`, and including the arguments `binwidth = 0.05` and `boundary = 0.4`. This creates the appropriate bins with boundaries at 0.3, 0.35, 0.4, 0.45, etc., as shown below:
```

File: 07-sampling.Rmd [hunk 10]
```diff
@@ -594,9 +561,10 @@ if (is_latex_output()) {
...
+Observe that all three histograms are bell-shaped and appear to center around a middle value, somewhere between 35% and 40%. In addition, as the sample size increases, the variation among the 1000 sample proportions decreases and there are fewer differences due to sampling variation.
+
+Moreover, the amount of sampling variation is actually proportional to the sample size used. To see this, we measure the amount of variation explicitly using a summary statistic called the \index{standard deviation} *standard deviation* (see Appendix \@ref(appendix-stat-terms) for a brief discussion on the properties of the standard deviation). The following data wrangling code introduces the `sd()` summary function to calculate the standard deviation of 1000 sample proportions of the same sample size. Therefore, three standard deviations are obtained, one for each sample size, as shown below.
```
Comment (ismayc) [inline 07-sampling.Rmd:567]: "the amount of sampling variation is actually proportional to the sample size used." This feels a little clunky and hard for a novice to decipher.

File: 07-sampling.Rmd [hunk 11]
```diff
@@ -633,21 +601,21 @@ comparing_n_table %>%
...
 
-- A. vary less,
-- B. vary by the same amount, or
-- C. vary more?
+- A. They vary less.
+- B. They vary by the same amount.
+- C. They vary more.
 
-**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** What summary statistic did we use to quantify how much the 1000 proportions red varied?
+**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** What summary statistic did we use to quantify how much the 1000 sample proportions varied?
```
Comment (ismayc) [inline 07-sampling.Rmd:618]: I think it is worth reminding readers what that "sample proportion" refers to here and maybe in a few other spots?

File: 07-sampling.Rmd [hunk 12]
```diff
@@ -663,111 +631,87 @@ As we observed in Figure \@ref(fig:comparing-sampling-distributions), as the sam
...
+In both our tactile and virtual sampling activities, we obtain samples in order to guess or *estimate* the proportion of red balls in the bowl. We use sampling for the purpose of *estimation* because is a less cumbersome approach than performing an exhaustive count of all the balls. The virtual sampling activity concludes with the results shown in Figure \@ref(fig:comparing-sampling-distributions) and Table \@ref(tab:comparing-n): comparing 1000 sample proportions of red balls using samples of sizes 25, 50, and 100. The conclusions of these activities can be considered our first attempt at understanding two key concepts relating to sampling for estimation:
```
Comment (ismayc) [inline 07-sampling.Rmd:634]:
```suggestion
In both our tactile and virtual sampling activities, we obtain samples in order to guess or *estimate* the proportion of red balls in the bowl. We use sampling for the purpose of *estimation*, because it is a less cumbersome approach than performing an exhaustive count of all the balls. The virtual sampling activity concludes with the results shown in Figure \@ref(fig:comparing-sampling-distributions) and Table \@ref(tab:comparing-n): comparing 1000 sample proportions of red balls using samples of sizes 25, 50, and 100. The conclusions of these activities can be considered our first attempt at understanding two key concepts relating to sampling for estimation:
```

File: 07-sampling.Rmd [hunk 13]
```diff
@@ -663,111 +631,87 @@ As we observed in Figure \@ref(fig:comparing-sampling-distributions), as the sam
...
 ```{r}
-virtual_shovel %>% 
+bowl %>%
+  rep_sample_n(size = 50) %>% 
   summarize(num_red = sum(color == "red")) %>% 
   mutate(prop_red = num_red / 50)
 ```
 
-So in our case, the value of `prop_red` is the **point estimate** of the population proportion $p$ since it estimates the latter's value. Furthermore, this point estimate has a specific name when considering proportions: the *sample proportion*. It is denoted using $\widehat{p}$ because it is a common convention in statistics to use a "hat" symbol to denote point estimates.
+The sample proportion, $\widehat{p}$, is given by the value of `prop_red`. It is also called a point estimate because is used to estimate the population proportion, $p$.
```
Comment (ismayc) [inline 07-sampling.Rmd:687]:
```suggestion
The sample proportion, $\widehat{p}$, is given by the value of `prop_red`. It is also called a point estimate because it is used to estimate the population proportion, $p$.
```

File: 07-sampling.Rmd [hunk 14]
```diff
@@ -794,32 +738,71 @@ In the upcoming Chapter \@ref(confidence-intervals) on confidence intervals, we'
...
+      strip.text = element_text(colour = "black"),
+      strip.background = element_rect(fill = "grey93")
+    )
+} else {
+  samp_distn_compare
+}
+```
+
+
+Moreover, observe that when the sample size used is 25 (left histogram) it is somewhat likely to obtain sample proportions of red balls that are less than 0.2 = 20% or greater than 0.55 = 55%; when the sample size is 50 (middle histogram) it is unlikely, although still possible, to get that extreme sample proportions; and when the sample size is 100 (right histogram) we have not observed a single sample proportion that is that extreme. As the sample size $n$ increases from 25 to 50 to 100, \index{sampling distributions!relationship to sample size} the variation of the sampling distribution decreases, and thus the values cluster more and more tightly around the same center between 35% and 40%. We quantified this variation using the standard deviation of our sample proportions in Table \@ref(tab:comparing-n), which we display again as Table \@ref(tab:comparing-n-repeat):
```
Comment (ismayc) [inline 07-sampling.Rmd:794]: Maybe three different sentences instead of connecting them with ;? Or turn this into bullet points for even more clarity?

File: 07-sampling.Rmd [hunk 15]
```diff
@@ -829,9 +812,11 @@ comparing_n_table %>%
...
+The standard deviation can be thought of "roughly" as how far, on average, the values are from the center of their distribution. In the context of sampling distributions, this is equivalent to thinking of how far, on average, a point estimate is from the population parameter it is trying to estimate since the parameter is equal to the center of the sampling distribution as shown in Figure \@ref(fig:comparing-sampling-distributions-3). Because of this relationship, the standard deviation of a sampling distribution is called the \index{standard error} **standard error** of a point estimate, where "error" can be thought of as the estimation error, on average, between a point estimate and the population parameter. For example, using \@ref(tab:comparing-n), the standard error for the sampling distribution of sample proportions when the sample size is 100 is 0.045 = 4.5%. If we were to take a random sample of 100 balls from the bowl, we expect the sample proportion of red balls to be, on average, 4.5% away from the proportion of red balls in the entire bowl.
```
Comment (ismayc) [inline 07-sampling.Rmd:815]:
```suggestion
The standard deviation can be thought of "roughly" as how far, on average, the values are from the center of their distribution. In the context of sampling distributions, this is equivalent to thinking of how far, on average, a point estimate is from the population parameter it is trying to estimate, since the parameter is equal to the center of the sampling distribution as shown in Figure \@ref(fig:comparing-sampling-distributions-3). Because of this relationship, the standard deviation of a sampling distribution is called the \index{standard error} **standard error** of a point estimate, where "error" can be thought of as the estimation error, on average, between a point estimate and the population parameter. For example, using \@ref(tab:comparing-n), the standard error for the sampling distribution of sample proportions when the sample size is 100 is 0.045 = 4.5%. If we were to take a random sample of 100 balls from the bowl, we expect the sample proportion of red balls to be, on average, 4.5% away from the proportion of red balls in the entire bowl.
```

File: 07-sampling.Rmd [hunk 16]
```diff
@@ -1235,7 +1178,7 @@ if (is_latex_output()) {
 }
 ```
 
-```{r echo=FALSE, purl=FALSE, results="asis"}
+```{r echo=FALSE, purl=FALSE, results="asis", eval = F}
```
Comment (ismayc) [inline 07-sampling.Rmd:1181]:
```suggestion
```{r echo=FALSE, purl=FALSE, results="asis", eval=FALSE}
```

**Comment by ismayc** [comment]
@avaldivi6 Just tagging you here so you get the notification

File: 07-sampling.Rmd [hunk 17]
```diff
@@ -35,26 +35,22 @@ options(scipen = 99, digits = 3)
 set.seed(76)
 ```
 
-In this chapter, we kick off the third portion of this book on statistical inference by learning about *sampling*. The concepts behind sampling form the basis of confidence intervals and hypothesis testing, which we'll cover in Chapters \@ref(confidence-intervals) and \@ref(hypothesis-testing). We will see that the tools that you learned in the data science portion of this book, in particular data visualization and data wrangling, will also play an important role in the development of your understanding.  As mentioned before, the concepts throughout this text all build into a culmination allowing you to "tell your story with data."
```
Comment (ismayc) [inline 07-sampling.Rmd:38]: Nice work on updating this!

File: 07-sampling.Rmd (hunk 6 above)
Comment (avaldivi6) [reply 07-sampling.Rmd:119]: Got it! See my new edits.

File: 07-sampling.Rmd (hunk 10 above)
Comment (avaldivi6) [reply 07-sampling.Rmd:566]: It will change once CLT is merged with it.

File: 07-sampling.Rmd (hunk 12 above)
Comment (avaldivi6) [reply 07-sampling.Rmd:633]: Done

File: 07-sampling.Rmd (hunk 14 above)
Comment (avaldivi6) [reply 07-sampling.Rmd:793]: Got it. But, it will change once CLT is merged with it.

File: 07-sampling.Rmd (hunk 15 above)
Comment (avaldivi6) [reply 07-sampling.Rmd:814]: Done

File: 07-sampling.Rmd [hunk 18]
```diff
@@ -35,26 +35,22 @@ options(scipen = 99, digits = 3)
...
... third portion of this book introduces statistical inference. This chapter is about *sampling*. Sampling involves drawing repeated random samples from a population and then using the information from those samples to learn more about the entire population. In Section \@ref(sampling-activity) we illustrate sampling using a tactile example and simultaneously introducing key concepts and terminology. In Section \@ref(sampling-simulation) we extend this by performing virtual sampling via simulations. The tools used in the data science portion of this book, in particular data visualization and data wrangling, continue to be useful in this context. In Section \@ref(sampling-framework) we introduce more definitions, terminology, and notation related to sampling. In Section \@ref(sampling-case-study) we tie the contents of this chapter to the real world by presenting a case study: a 2013 poll on then U.S. President Barack Obama's popularity among young Americans, conducted by Kennedy School's Institute of Politics at Harvard University. Finally, we generalize *sampling* to many scenarios and use sampling to illustrate one of the fundamental theoretical results: the *Central Limit Theorem*.
```
Comment (ismayc) [inline 07-sampling.Rmd:38]: @avaldivi6 We need to update this to remove the Obama poll reference?

File: 07-sampling.Rmd [hunk 19]
```diff
@@ -64,137 +60,119 @@ library(scales)
...
 ```
 
 
-```{r sampling-exercise-3, echo=FALSE, fig.cap="Removing 50 balls from the bowl.", purl=FALSE, out.width = "100%", purl=FALSE}
+```{r sampling-exercise-3, echo=FALSE, fig.cap="Taking a sample of 50 balls from the bowl.", purl=FALSE, out.width = "100%", purl=FALSE}
 include_graphics("images/sampling/balls/sampling_bowl_3_cropped.jpg")
 ```
 
-Observe that 17 of the balls are red and thus 0.34 = 34% of the shovel's balls are red. We can view the proportion of balls that are red in this shovel as a guess of the proportion of balls that are red in the entire bowl. While not as exact as doing an exhaustive count of all the balls in the bowl, our guess of 34% took much less time and energy to make. 
+Observe that 17 of the balls are red and thus the proportion of red balls in the sample is 17/50 = 0.34 or 34%. Think of the proportion or red balls in the sample as a guess of the proportion of red balls in the entire bowl. It is likely that the proportion of red balls in the entire bowl is not exactly 34% but, as we show later, this is probably a good guess and it took much less time and energy to obtain.
```
Comment (ismayc) [inline 07-sampling.Rmd:95]:
```suggestion
Observe that 17 of the balls are red and thus the proportion of red balls in the sample is 17/50 = 0.34 or 34%. Think of the proportion of red balls in the sample as a guess of the proportion of red balls in the entire bowl. It is likely that the proportion of red balls in the entire bowl is not exactly 34% but, as we show later, this is probably a good guess and it took much less time and energy to obtain.
```

File: 07-sampling.Rmd [hunk 20]
```diff
@@ -64,137 +60,119 @@ library(scales)
...
+### Recap, some terminology, and additional remarks {#sampling-what-did-we-just-do}
 
-Moreover, because we mixed the balls before each use of the shovel, the samples were randomly drawn. Because each sample was drawn at random, the samples were different from each other. Because the samples were different from each other, we obtained the different proportions red observed in Figure \@ref(fig:samplingdistribution-tactile). This is known as the concept of *sampling variation*. \index{sampling!variation}
+The activity introduced the concept of \index{sampling} *sampling*. We wanted to know the proportion of red balls in a bowl. Since the bowl had many balls, performing an exhaustive count of red and white balls was impractical. Instead, we first took a *sample* of 50 balls and determine that the proportion of red balls in this sample was 34%. For simplicity, we say the *sample proportion* was 34%. This is an *estimate* of the proportion of red balls in the entire bowl. We then took more samples and obtain more sample proportions, 33 in total. This activity is what in statistics we call *sampling*.
```
Comment (ismayc) [inline 07-sampling.Rmd:160]:
```suggestion
The activity introduced the concept of \index{sampling} *sampling*. We wanted to know the proportion of red balls in a bowl. Since the bowl had many balls, performing an exhaustive count of red and white balls was impractical. Instead, we first took a *sample* of 50 balls to determine that the proportion of red balls in this sample was 34%. For simplicity, we say the *sample proportion* was 34%. This is an *estimate* of the proportion of red balls in the entire bowl. We then took more samples to obtain more sample proportions, 33 in total. This activity is what in statistics we call *sampling*. 
```

File: 07-sampling.Rmd [hunk 21]
```diff
@@ -462,26 +435,18 @@ A shovel with 25 slots          |  A shovel with 50 slots  | A shovel with 100 s
...
-
-Using our newly developed tools for virtual sampling, let's unpack the effect of having different sample sizes! In other words, let's use `rep_sample_n()` with `size` set to `25`, `50`, and `100`, respectively, while keeping the number of repeated/replicated samples at 1000:
-
-1. Virtually use the appropriate shovel to generate 1000 samples with `size` balls.
-1. Compute the resulting 1000 replicates of the proportion of the shovel's balls that are red.
-1. Visualize the distribution of these 1000 proportions red using a histogram.
-
-Run each of the following code segments individually and then compare the three resulting histograms.
+We follow the same process performed in subsection \@ref(shovel-1000-times): we generate 1000 samples, obtain the sample proportions, and use them to draw a histogram. We do this three times with the `size` argument set to `25`, `50`, and `100`, respectively. Run each of the following code segments individually and then compare the resulting histograms.
```
Comment (ismayc) [inline 07-sampling.Rmd:438]:
```suggestion
We follow the same process performed in Subsection \@ref(shovel-1000-times): we generate 1000 samples, obtain the sample proportions, and use them to draw a histogram. We do this three times with the `size` argument set to `25`, `50`, and `100`, respectively. Run each of the following code segments individually and then compare the resulting histograms.
```

File: _bookdown.yml
Comment (ismayc) [inline _bookdown.yml:1]: @avaldivi6 Can you revert the changes made here to not have these files commented out?

File: 07-sampling.Rmd [hunk 22]
```diff
@@ -904,66 +894,24 @@ Remember the key message of this last table: that as the sample size $n$ goes up
...
-
-1. The effect of *sampling variation* on our estimates.
-1. The effect of sample size on *sampling variation*.
+Observe that taking 1000 repeated random samples of different sizes was useful to illustrate how the sampling distribution of point estimates works, and what are the best practices to obtain accurate and precise estimates. In a real-life situation when getting the best estimate is the main task, we try to get a single random sample that is as large as we can afford because we know that the resulting point estimate will be the most accurate and precise. 
 
-This is not how sampling is done in real life! In a real-life scenario, we wouldn't take 1000 repeated/replicated samples, but rather a single sample that's as large as we can afford. In Section \@ref(sampling-case-study), we're going to study a real-life example of sampling: polls.
+In Section \@ref(sampling-case-study), we're going to study a real-life example of sampling: polls.
```
Comment (ismayc) [inline 07-sampling.Rmd:914]: @avaldivi6 Need to update here too

File: 07-sampling.Rmd [hunk 23]
```diff
@@ -1068,7 +1016,7 @@ Eighth, is the sampling procedure **unbiased**? In other words, do all observati
 Ninth and lastly, was the sampling done at **random**? \index{sampling!random}
 
 * Bowl: As long as you mixed the bowl sufficiently before sampling, your samples would be random.
-* Obama poll: Was the sample conducted at random? We can't answer this question without knowing about the *sampling methodology*\index{sampling methodology} used by Kennedy School's Institute of Politics at Harvard University. We'll discuss this more at the end of this section.
+* Obama poll: Was the sample conducted at random? We can't answer this question without knowing about the *sampling methodology*\index{sampling methodology} used by Kennedy School's Institute of Politics at Harvard University. We will discuss this more at the end of this section.
```
Comment (ismayc) [inline 07-sampling.Rmd:1019]: @avaldivi6 Need to remove this section on the Obama poll?

File: 07-sampling.Rmd [hunk 24]
```diff
@@ -1122,33 +1070,355 @@ Comment on the representativeness of the following *sampling methodologies*:
...
 
-```{r CLT-video-preview, echo=FALSE, fig.cap="Preview of Central Limit Theorem video.", purl=FALSE, out.width = "75%"}
-knitr::include_graphics("images/copyright/CLT_video_preview.png")
+* Bowl: $N$ = `r num_balls` identically sized red and white balls
+* Gallup poll: $N$ = ? Americans aged 18 or older
+
+Second, what is the **population parameter**? \index{sampling!population parameter}
+
+* Bowl: The population proportion $p$ of *all* the balls in the bowl that are red.
+* Gallup poll: The population proportion $p$ of *all* Americans who do not believed that pre-pandemic normalcy was attainable.
```
Comment (ismayc) [inline 07-sampling.Rmd:1092]:
```suggestion
* Gallup poll: The population proportion $p$ of *all* Americans who do not believe that pre-pandemic normalcy was attainable.
```

File: 07-sampling.Rmd [hunk 25]
```diff
@@ -1122,33 +1070,355 @@ Comment on the representativeness of the following *sampling methodologies*:
...
+
+Fourth, how do you perform **sampling** to obtain a sample of size $n$? \index{sampling}
+
+* Bowl: Using a shovel with $n$ slots. 
+* Gallup poll: One method is to get a list of phone numbers of all Americans, 18 and older, and pick out $n$ phone numbers. In this poll's case, the sample size of this poll was $n = 1,000$.
+
+Fifth, what is your **point estimate** also known as the **sample statistic** of the unknown population parameter?
+
+* Bowl: The sample proportion $\widehat{p}$ of red balls. 
+* Gallup poll: The sample proportion $\widehat{p}$ of Americans in the sample that approve of Obama's job performance. In this poll's case, $\widehat{p} = 0.41 = 41\%$, the quoted percentage in the second paragraph of the article. \index{point estimate} \index{sample statistic}
```
Comment (ismayc) [inline 07-sampling.Rmd:1107]: @avaldivi6 This needs to be updated to remove Obama.

File: 07-sampling.Rmd [hunk 26]
```diff
@@ -1235,15 +1505,15 @@ if (is_latex_output()) {
 }
 ```
 
-```{r echo=FALSE, purl=FALSE, results="asis"}
+```{r echo=FALSE, purl=FALSE, results="asis", eval = FALSE}
```
Comment (ismayc) [inline 07-sampling.Rmd:1508]: @avaldivi6 I'm pretty sure this does need to run just in case things move around?

File: 07-sampling.Rmd [hunk 27]
```diff
@@ -1235,15 +1505,15 @@ if (is_latex_output()) {
...
 generate_r_file_link("07-sampling.R")
 ```
 
 
 ### What's to come?
 
-Recall in our Obama poll case study in Section \@ref(sampling-case-study) that based on this particular sample, the best guess by Kennedy School's Institute of Politics at Harvard University of the U.S. President Obama's approval rating among all young Americans was 41%. However, this isn't the end of the story. If you read the article further, it states:
+Recall in our Gallup poll case study in Section \@ref(sampling-case-study) that based on this particular sample, the best guess by Kennedy School's Institute of Politics at Harvard University of the U.S. President Obama's approval rating among all young Americans was 41%. However, this isn't the end of the story. If you read the article further, it states:
```
Comment (ismayc) [inline 07-sampling.Rmd:1515]: Update needed

**Comment by ismayc** [comment]
Plan is for Arturo to make all needed updates on his fork for Chapter 7, and then we'll do a final review here. I'll plan to review each incremental update on his fork.
